tools/m2-repo-cleaner/pom.xml

```diff
@@ -0,0 +1,47 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.fusesource.ide</groupId>
+		<artifactId>tools</artifactId>
+		<version>10.0.0-SNAPSHOT</version>
+		<relativePath>..</relativePath>
+	</parent>
+
+	<artifactId>m2-repo-cleaner</artifactId>
+	<name>JBoss Fuse Tooling :: Tools :: Local Repository Repair Tool</name>
+	<description>IDE m2 repair tools</description>
+
+	<prerequisites>
+		<maven>2.0.9</maven>
+	</prerequisites>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.0</version>
+			</plugin>
+
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>exec-maven-plugin</artifactId>
+				<version>1.2.1</version>
+				<executions>
+					<execution>
+						<goals>
+							<goal>java</goal>
+						</goals>
+					</execution>
+				</executions>
+				<configuration>
+					<mainClass>org.fusesource.ide.tools.mavenutils.M2RepoChecker</mainClass>
+					<cleanupDaemonThreads>false</cleanupDaemonThreads>
```

Comment:
isn't dangerous to not clean daemon threads? we might pollute the CI with daemon threads